### PR TITLE
Google analytics

### PR DESCRIPTION
**Pull request by Mar1nka**

### CODE REVIEW

File: src/client/helpers/analytics.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import ReactGA from 'react-ga';
+
+const GA_TRACKING_ID = 'UA-XXXXX-Y';
```
Comment (SobakaSlava) [inline src/client/helpers/analytics.js]: This variable should be placed in [config](https://github.com/paralect/nextjs-landing-starter/tree/4958fc12e4f903ddf2aae92235c6c12271693ba1/src/server/config/environment)
[Here](https://github.com/paralect/nextjs-landing-starter/blob/4958fc12e4f903ddf2aae92235c6c12271693ba1/src/client/helpers/api/api.client.js#L2) is example of how to take it from config

Comment (Mar1nka) [reply src/client/helpers/analytics.js]: @SobakaSlava Where exactly in the config?

Comment (SobakaSlava) [reply src/client/helpers/analytics.js]: Near the `apiUrl` variable

File: src/server/config/environment/development.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
   apiUrl: 'http://localhost:3001',
   webUrl: 'http://localhost:3002',
+  GA_TRACKING_ID: 'GA_TRACKING_ID',
```
Comment (IharKrasnik) [inline src/server/config/environment/development.js]: @Mar1nka please rename this variable to match our styles and prevent issues with linter in the future (`GA_TRACKING_ID` => `gaTrackingId`)

File: src/server/config/next.config.js [hunk 3]
```diff
@@ -2,7 +2,12 @@ const { resolve } = require('path');
 
 const withCSS = require('../lib/next-css');
 
-const { apiUrl, webUrl, isDev } = require('./index');
+const {
+  apiUrl,
+  webUrl,
+  isDev,
+  GA_TRACKING_ID,
```
Comment (IharKrasnik) [inline src/server/config/next.config.js]: Same here

**Pull request merged**